test/primitives/array-element-test.js

```diff
@@ -176,6 +196,18 @@ describe('ArrayElement', function() {
         arrayElement.push('foobar');
         itAddsToArray(arrayElement);
       });
+
+      it('sets the parent to children', function() {
+        var item = new minim.StringElement('foobar');
+        arrayElement.push(item);
+        expect(item.parents).to.include(arrayElement);
```

Comment:
Seems this functionality should be private and associated only with event functionality if that is really the purpose of parents here. Not clear.